app/autoload.php

```diff
@@ -3,8 +3,8 @@
 use Doctrine\Common\Annotations\AnnotationRegistry;
 use Composer\Autoload\ClassLoader;
 
-/**
- * @var ClassLoader $loader
+/*
+ * @var ClassLoader
```

Comment:
-1 for this change.

The old version was a completely valid doc block for the `$loader` variable. Seems like it isn't build into the PHP CS fixer yet.

The new version doesn't make any sense, as it's no longer a doc block and just a normal comment. No IDE will read it as being a docblock